src/Client.php

```diff
@@ -52,20 +67,21 @@ function __construct(ClientInterface $client, $baseUrl)
      * @param array $options guzzle options
      * @return ResponseInterface
      * @throws GuzzleException
-     * @throws ValidationException
+     * @throws ValidationException|CertificateException
      */
     protected function sendRequest($method, $endpoint, array $options = [])
     {
         try {
-            return $this->client->request($method, $this->baseUrl . $endpoint, array_merge([
```

Comment:
This is not good. If you remove the base URL you force the user to into a specific implementation of `ClientInterface` (guzzle in this case) where that URL can be set elsewhere.